.pylintrc

```diff
@@ -61,7 +61,8 @@ disable=fixme,
         old-style-class,
         superfluous-parens,
         bad-continuation,
-        line-too-long
+        line-too-long,
+        wrong-import-order
```

Comment:
Is this necessary? I kind of like the rule to keep imports organized...

Reply:
I was having issues with pylint 1.9.4 (py27) vs pylint 2.3.1 (py37) ordering expectations when using `from x import y as z`, but not doing that anymore, attempting revert.